lib/puppet/type/cisco_ace.rb

```diff
@@ -291,4 +291,12 @@ def name
     desc 'Log matches against this entry'
     newvalues(:true, :false)
   end
+
+  validate do
+    unless self[:remark].nil?
+      fail ArgumentError,
+           'established and log parameters must be nil when remark is not nil' unless
```

Comment:
@shermdog Interested in your opinion here.  The only reason we are adding this validate check is because prior to this change the output of the `puppet resource` command would display these two properties set to `false` even though they are not used for the remark ace.  Older manifests will error out with a strange error message so this validate provides more meaningful guidance.